client/src/components/existing-request/existing-request-display.vue

```diff
@@ -310,15 +310,20 @@ export default class ExistingRequestDisplay extends Mixins(
 
   private get reviewDate () {
     if (this.nr.waiting_time) {
-      let waitingTime = this.nr.waiting_time
+      let queueTime = this.nr.waiting_time
+      let submittedDate = Moment(this.nr.submittedDate)
+      let todaysDate = Moment()
+      // get number of days since the nr was submitted
+      let diffDays = todaysDate.diff(submittedDate, 'days')
+      let waitingTime = queueTime - diffDays + 1
```

Comment:
this makes it a day long
let waitingTime = queueTime - diffDays + 1

let waitingTime = Math.max(queueTime - diffDays, 1)
same logic then as the wait time
and lines 314-317 can be removed

Reply:
Done